backend/node_app/controllers/userController.js

```diff
@@ -481,6 +482,25 @@ class UserController {
 		}
 	}
 
+	async sendClassificationAlert(req, res) {
+		let userId = 'Unknown_Webapp';
+		try {
+			userId = req.get('SSL_CLIENT_S_DN_CN');
```

Comment:
Hash the userId here same as how we store it hashed in PG that way we don't send out sensitive info.